src/modules/esl-footnotes/core/esl-note.ts

```diff
@@ -114,11 +114,17 @@ export class ESLNote extends ESLBaseElement {
       this._$footnotes?.turnOffHighlight(this);
     }
     if (attrName === 'ignore') {
-      memoize.clear(this, 'queryToIgnore');
-      this.$$on(this._onBPChange);
+      this.updateIgnore();
     }
   }
 
+  /** Revise the settings for ignoring the note */
+  public updateIgnore(): void {
```

Comment:
The only thing I'm concerned about is the name of that function, especially in the context of its public.
TBD: @exadel-inc/esl-core-team to discuss naming

Reply:
I also hesitated with that name.
There was a great desire to call this method something like this `reviseIgnore()`

Reply:
```suggestion
  public updateIgnoredQuery(): void {
```